bedrock/mozorg/urls.py

```diff
@@ -293,11 +293,12 @@
     page('internet-health/web-literacy', 'mozorg/internet-health/web-literacy.html'),
     page('internet-health/decentralization', 'mozorg/internet-health/decentralization.html'),
 
-    page('moss', 'mozorg/moss/index.html'),
+    url(r'moss/$', views.moss_view, name='mozorg.moss'),
```

Comment:
I believe this should be `r'^moss/$'` else any URL that ends with `moss` would be valid? (e.g. `foo/moss/`)

Reply:
D'oh! Good catch.